src/vs/workbench/contrib/welcomeOverlay/browser/welcomeOverlay.css

```diff
@@ -0,0 +1,16 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
this should go into the existing src/vs/workbench/contrib/welcomeOverlay/browser/media/welcomeOverlay.css